projects/assets-library/assets/styles/_font.scss

```diff
@@ -329,3 +330,7 @@ $font-family: 'Work Sans', sans-serif;
 @mixin axis-title($color: $gray-9) {
   @include chart-small-regular;
 }
+
+$mat-typography: mat.define-typography-config(
```

Comment:
Did this come on its own with migration

Reply:
I suggested this from reading the docs - looks like they have some form of migration for it, but not sure exactly what conditions trigger it or if we just skipped it: https://github.com/angular/components/pull/22698